subprojects/core/src/main/java/org/gradle/initialization/buildsrc/BuildSrcUpdateFactory.java

```diff
@@ -42,6 +43,7 @@ public BuildSrcUpdateFactory(BuildController buildController, BuildSrcBuildListe
     }
 
     @Override
+    @Nonnull
```

Comment:
This shouldn't be necessary given the package is now `@NonNullApi`

Reply:
It's overriding a `@Nullable` method so the linter assumes `@Nullable` as well. The explicit `@NonNullApi` lets the linter know that this particular version never returns null.